script.js

```diff
@@ -0,0 +1,37 @@
+var choice = "";
+var type = prompt("You're finally old enough to choose your first Pokemon! Professor Oak gives you 3 pokeballs to choose from. What type of Pokemon do you choose? Water, Fire or Grass (W, F, G)?");
+if (choice == "W" || "Water") {
+  alert("Your first Pokemon is Squirtle!");
+}
```

Comment:
I already talked to you about this, but the string `"Water"`  is truthy, therefore no matter what the user inputs it will always return true and the first condition will always be met.